content/docs/reference/clouds/azure/_index.md

```diff
@@ -58,6 +58,7 @@ The Azure provider is open source and available in the [pulumi/pulumi-azure](htt
 The Azure provider accepts the following configuration settings.  These can be provided to the default Azure provider via `pulumi config set azure:<option>`, or passed to the constructor of `new azure.Provider` to construct a specific instance of the Azure provider.
 
 * `environment`: (Required) The cloud environment to use. It can also be sourced from the ARM_ENVIRONMENT environment variable. Supported values are: `public` (default), `usgovernment`, `german`, `china`.
+* `location`: (Required) The location to use. ResourceGroups will consult this property for a default location, if one was not supplied explicitly.
```

Comment:
I also went ahead and added this so users are aware of this feature. I think it is super useful for Azure users. I assume we don't auto-generate this page?

Reply:
It is optional, right? (It says "(Required)".)